crates/rpc/rpc/Cargo.toml

```diff
@@ -82,3 +83,15 @@ jsonrpsee-types.workspace = true
 jsonrpsee = { workspace = true, features = ["client"] }
 assert_matches.workspace = true
 tempfile.workspace = true
+
+[features]
+optimism = [
```

Comment:
this does add the feature?

Reply:
yeah, I could just remove the usage of the feature, but to fix broken tests, the deps need to be pulled in with the feature